astro/src/content/docs/lifecycle/manage-users/verification/identity-pre-verification-via-api.mdx

```diff
@@ -0,0 +1,206 @@
+4. Create the user using the returned verificationId
+
+If you're looking for the hosted pages version with Advanced Registration Forms, see:
+
+- [Email Identity Pre-Verification](/docs/lifecycle/manage-users/verification/mail-identity-verification)
+- [Phone Identity Pre-Verification](/docs/lifecycle/manage-users/verification/phone-identity-verification)
+
+## Prerequisites
+
+Before you begin, ensure your environment meets these requirements. This guide assumes you are running FusionAuth version 1.62.0 or greater. If you aren't, head on over to the [installation guide](/docs/get-started/download-and-install) or the [upgrade guide](/docs/operate/deploy/upgrade) to update to a version with support for this feature.
+
+<Aside type="note">
+  Email/Phone Verification and identity pre-verification are not premium features; however, enforcing pre-verification with Advanced Registration Forms as well as Gating is.
+  If you are following this guide implementing pre-verification entirely via API, you can still build equivalent gating in your application logic.
+  See the hosted pages guides if you prefer FusionAuth to enforce gating for you.
+</Aside>
+
+## General Flow
+
+At a glance, the identity pre-verification process follows these high-level steps.
+
+1. Start Verification Flow
+   - POST /api/identity/verify/start
+   - Provide the identity value in <InlineField>loginId</InlineField> and the type in <InlineField>loginIdType</InlineField> (`email` or `phoneNumber`).
+   - For pre-verifying identities before user creation, set <InlineField>existingUserStrategy</InlineField> to `mustNotExist`.
+   - The response includes a <InlineField>verificationId</InlineField> and may include a <InlineField>oneTimeCode</InlineField> if using the `FormField` strategy.
+
+2. Send Code (optional)
+   - POST /api/identity/verify/send
+   - Provide the <InlineField>verificationId</InlineField>. Use this if you want FusionAuth to send the email or SMS containing the code or clickable link.
+
+3. Complete Verification
+   - POST /api/identity/verify/complete
+   - Provide the <InlineField>verificationId</InlineField> and, if using the `FormField` strategy, the <InlineField>oneTimeCode</InlineField>.
+   - If the strategy is `ClickableLink`, you can validate using just the <InlineField>verificationId</InlineField> (the user clicks the link).
+
+4. Create the User
+   - POST /api/user or /api/user/registration
+   - Include the <InlineField>verificationIds</InlineField> array from step 1 associated to the verified identity.
+
+<Aside type="note">
+  <InlineField>existingUserStrategy</InlineField> values are lowercase: `mustExist` and `mustNotExist`.
+
+  The default <InlineField>verificationStrategy</InlineField> depends on your tenant configuration (for email: `tenant.emailConfiguration.verificationStrategy`; for phone: `tenant.phoneConfiguration.verificationStrategy`), and may be either `ClickableLink` or `FormField`.
```

Comment:
```suggestion
  The default <InlineField>verificationStrategy</InlineField> depends on your tenant configuration:
  
  - email: `tenant.emailConfiguration.verificationStrategy`
  - phone: `tenant.phoneConfiguration.verificationStrategy`, either `ClickableLink` or `FormField`
```
[suggestion]: to make this easier to read, use bullet points, even though it's only two items. with long fields like this, it's a lot easier to compare them when they're (relatively) vertically aligned in bullets.
I'm not sure which clause the 'and may be either' phrase modifies, so we should clean that up regardless to avoid ambiguity.